src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java

```diff
@@ -74,8 +77,20 @@ public CommandResult execute(Model model) throws CommandException {
         // Create edited appointment
         Appointment appointmentToEdit = appointmentList.get(zeroBasedAppointmentIndex);
         Person patient = appointmentToEdit.getPerson();
+        Doctor targetDoctor = null;
```

Comment:
Is it possible to change the editing of appointment's doctor to be by index? Since our add appointment command is by index also. Otherwise, write a message to tell users it is not by index.